pylint_odoo/test_repo/no_odoo_module/eval_used.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+
+
+def func2(param):
```

Comment:
Can we name these methods something a little more descriptive? I know they're test methods, but `func2` and `func3` really hold no context on their own and require that you read the methods, or at least the doc block, to understand what they are doing.